packages/bundlers/experimental/src/ExperimentalBundler.js

```diff
@@ -329,7 +329,9 @@ function createIdealGraph(
   let bundleRootGraph: ContentGraph<
     BundleRoot | 'root',
     $Values<typeof bundleRootEdgeTypes>,
-  > = new ContentGraph();
+  > = new ContentGraph({
+    edgeTypes: dependencyPriorityEdges,
```

Comment:
@AGawrys This new `edgeTypes` property needs to list all possible edge types for a graph. Is `dependencyPriorityEdges` an exhaustive list of edge types for the experimental bundler?